launchdarkly/resource_feature_flag.go

```diff
@@ -43,6 +45,35 @@ func resourceFeatureFlag() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"variations_kind": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "boolean",
+				ValidateFunc: validateFeatureFlagVariationsType,
+				ForceNew:     true,
+			},
+			"variations": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MinItems: 2,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"value": {
+							Type:     schema.TypeString,
```

Comment:
Check for empty strings (all three attributes)

Reply:
Is it Terraform good pratice ?

Theoretically sending an empty string to the launchdarkly API is valid :

```
    {
      value = "allo3"
      name = ""
      description = ""
    }
```
This configuration would be valid, that's why name and description are set with optional=true

Reply:
Oh it's fine then. Accept empty strings then